JDTeamAcetabulum/qna
# Instructors can save or publish questions

In this commit, Instructor are allowed to either save or publish questions. if instructor save the question, only other instructor can view the question. student can view the question only if instructor publish the question. the published date and time is noted and shown in the question tab.



## Changes 
- Added an ability to Publish a question. 
## Context 
- When Instructor is adding the question 

## Testing 
- Create a question and save it. Check published date (should be empty because it is not published yet)
- Create a question and publish it. Check the published date
- Edit the saved question and publish it. Check the published date

## Pre-merge checklist 

- [x] Build succeeds locally and in CI
- [x] Proper documentation
- [x] Git history clean
